prototypes/form-wizard/confirmation.html

```diff
@@ -1,5 +1,7 @@
 ---
 layout: page
+permalink: /confirmation/
+title: Thank you for your submission
```

Comment:
some experimentation locally shows that omitting the `title` but keeping the `permalink` is the compromise we want. The `title` puts it in the nav bar, which we don't want.